indigo_app/templates/library.html

```diff
@@ -37,74 +37,75 @@
   <thead>
     <tr>
       <th class="title" data-sort="title">Title</th>
-      <th class="year" data-sort="year">Year / Number</th>
+      <th class="uri" data-sort="uri">FRBR URI</th>
+      <th class="langs" data-sort="langs">Languages</th>
```

Comment:
We need to ensure that `langs` and `psit` are then fields on the `work` object, somewhere before line 397 in library.js